Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:latest
```

у нас 14 версия, лучше указать нашу реальную 
```suggestion
FROM node:14
```

---

Тогда 16) Поправлю